.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/The-OpenROAD-Project/yosys.git
 [submodule "tools/OpenROAD"]
 	path = tools/OpenROAD
-	url = ../OpenROAD.git
+	url = https://github.com/The-OpenROAD-Project/OpenROAD.git
```

Comment:
@louiic is this change really needed?

Reply:
Yes ... without this, the fork can never find OpenROAD GIT

Reply:
@louiic It took me a little while to remember why this was important. The reason it was done with ../ is so that a clone from the -private project would clone the OpenROAD submodule from the -private project. Also clones from forks will clone the forked openroad submodule. In your case, if you made a fork of openroad as well as openroad-flow-scripts you should be able to run without this change. The ../ is not strictly required but for the users of forks and -private its confusing when ORFS comes from their fork but OR does not.